cyclops-ctrl/internal/prometheus/handler.go

```diff
@@ -33,3 +87,27 @@ func (m *Monitor) IncModule() {
 func (m *Monitor) DecModule() {
 	m.ModulesDeployed.Dec()
 }
+
+func (m *Monitor) UpdateCacheMetrics(cache *ristretto.Cache) {
+	cacheMetrics := cache.Metrics
+
+	m.CacheHits.Add(float64(cacheMetrics.Hits()))
```

Comment:
Shouldn't we set cacheMetrics values instead of adding them since cacheMetrics will give you the exact value of the metric, not just the diff? This also requires to update metrics from counters to gauges

Reply:
resolved in: 43b5a46f6d45079f5ec86c80487a33b6e5c5893d

- changed the cache metric type in monitor to `prometheus.Gauge`
- refactored to set the latest cache metric instead of adding to the cache metrics